### PR TITLE
made basic startgame page

### PR DESCRIPTION
**Pull request by casesensative**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 # misc
 .DS_Store
+.env
```
Comment (michellegaletti) [inline .gitignore:18]: .env was added to .gitignore already on line 3, don't need it here again

**Review by michellegaletti — CHANGES_REQUESTED** [review]
Fix your .gitignore file by deleting the one line and it's ready for a merge! Awesome work!

File: .gitignore (hunk 1 above)
Comment (michellegaletti) [reply .gitignore:18]: Will add in as bug.

**Review by michellegaletti — APPROVED** [review]

**Review by asalduur — APPROVED** [review]
👍🏾

**Pull request merged**